docker-assets/entrypoint

```diff
@@ -5,24 +5,25 @@ function urlescape() {
 }
 
 if [[ ! -z "${ACG_CONFIG}" ]]; then
-  export DATABASE_HOST=`jq -r '.database.hostname' ${ACG_CONFIG}`
-  export DATABASE_PORT=`jq -r '.database.port' ${ACG_CONFIG}`
-  DATABASE_USER=`jq -r '.database.username' ${ACG_CONFIG}`
-  DATABASE_PASSWORD=`jq -r '.database.password' ${ACG_CONFIG}`
+  export DATABASE_HOST=ClowderConfig.instance["databaseHostname"]
```

Comment:
@hsong-rh I don't think this is going to work because this is a shell script and we won't have the Ruby ClowderConfig instance available